test/models/host/subscription_facet_test.rb

```diff
@@ -215,6 +215,71 @@ def test_update_foreman_facts
       assert_includes values.map(&:name), '_timestamp'
     end
 
+    def test_update_foreman_facts_with_centos_no_minor_version
+      host.operatingsystem = ::Operatingsystem.create(
```

Comment:
Take a look and see if we have a fixture we could use for the operatingsystem, or perhaps a factorybot factory (it might exist in foreman)

Reply:
There are foreman fixtures/factories, it appears.